### PR TITLE
Parser for the FLEUR (MaX_R5) `banddos.hdf`  file

### PR DESCRIPTION
**Pull request by anoopkcn**
The file contains, utility functions for inspecting and parsing `bahddos.hdf` file

### CODE REVIEW

**Comment by PhilippRue** [comment]
Maybe a Fleur user (@janssenhenning or @Tseplyaev) could take a look and merge this in?

File: masci_tools/io/parsers/banddos.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    kpts[0] = np.linalg.norm(reciprocalCell.dot(kcoord[0]))
+    for i in range(1, nkpt):
+        kpts[i] = kpts[i - 1] + \
+            np.linalg.norm(reciprocalCell.dot(
+                kcoord[i]) - reciprocalCell.dot(kcoord[i - 1]))
+    return kpts
+
+
+def writeBand(inpFile, outFile1, outFile2):
+    eV = 27.211386
```
Comment (janssenhenning) [inline masci_tools/io/parsers/banddos.py:60]: We now have a module `masci_tools.util.constants`, where we define all the constants that should be used in masci-tools and the aiida-plugins

Comment (janssenhenning) [reply masci_tools/io/parsers/banddos.py:60]: You can use `masci_tools.util.constants.HTR_TO_EV` instead

**Review by janssenhenning — COMMENTED** [review]
So if I understand correctly this is a utility to convert the `banddos.hdf` file to a easy-to-plot `.txt` file specifically for bandstructures, right?

For the moment this is of course very useful. I hope that we can have a more general HDF5 parser very soon, which can be configured for different files or DOS calculations.

File: masci_tools/io/parsers/banddos.py (hunk 1 above)
Comment (anoopkcn) [reply masci_tools/io/parsers/banddos.py:60]: A the moment there are only a few functions in the file, for inspecting the `banddos.hdf` and to write the band to a file.  If there is demand I can convert this into a generalised parser Class for parsing the `hdf`.

**Comment by anoopkcn** [comment]
I think it is a good idea to write a generalised version of this. So keep the pull request open I will commit to it.

**Comment by janssenhenning** [comment]
@anoopkcn Maybe we should meet shortly, because I was also thinking about this and collected some ideas

**Comment by anoopkcn** [comment]
> @anoopkcn Maybe we should meet shortly, because I was also thinking about this and collected some ideas
Yes, I'm free in the afternoon, we shall continue the conversation in iffchat

**Comment by janssenhenning** [comment]
@chand If you're okay with it I will close this in favor of #30 and merge that PR.

**Comment by anoopkcn** [comment]
Please close the issue since the new branch has a better solution. Also, the user `chand` is a different person on github :) I think she is getting all the notification instead of me.

**Comment by janssenhenning** [comment]
> Also, the user `chand` is a different person on github :) I think she is getting all the notification instead of me.

Yeah noticed it too late (Sorry if you're reading this :D)

**Pull request closed without merging**